README.md

```diff
@@ -12,6 +12,15 @@ On Linux it's possible to start TruckersMP through [Proton][github:proton]. A wo
 ## Known issues
 
 * It was reported that "ProMods + ETS2MP" crashes with Proton 5.13 but Proton 5.0 works ([issue #147][github:issue147])
+* If (default) OpenGL rendering backend is used, game crashes due to a bug in Multiplayer DLL when trying to choose a color for "Player tag" or "Players on the world map" ([issue #91][github:issue91])
+    * A workaround is to use D3D11 rendering backend by specifying `-d` (`--enable-d3d11`)
+* If D3D11 rendering backend is used, TruckersMP login screen is not shown without Windows native 64-bit `d3dcompiler_47.dll`
+    * `truckersmp-cli` provides `--activate-native-d3dcompiler-47` option as a workaround: When this is specified with `-s` (`--start`), `truckersmp-cli` downloads/activates the DLL
+        * Once the DLL is activated, there's basically no need to specify the option again
+        * When downgrading Proton, the native DLL is removed: Then `--activate-native-d3dcompiler-47` is needed again
+* When DXVK is used in fullscreen mode, TruckersMP overlay disappears after switching window (e.g. pressing Alt + Tab) ([issue #90][github:issue90])
```

Comment:
Now I'm using Mesa 21.0.0-rc5 (RADV) + Proton 5.13-6 and can't reproduce the "Alt-Tab" issue anymore. Unfortunately I don't have other working machines or graphics cards and can't test non-AMD GPUs.
```suggestion
* When DXVK is used in fullscreen mode, TruckersMP overlay may disappear after switching window (e.g. pressing Alt + Tab) ([issue #90][github:issue90])
```

Reply:
Today I tested Proton 5.0 and 4.11 (with old DXVK versions) and could not reproduce it. So I guess recent DXVK didn't fix this issue.

If this is a fix in Multiplayer DLL, we can close that issue, remove the notes, and get rid of `--wine-desktop` option.

We need to collect more information.

Reply:
Also with Proton 5.13 and nvidia it's working. Guess we can remove that note and[ we probably could now make dxvk the default.](https://github.com/lhark/truckersmp-cli/pull/105#issuecomment-699059693)

Reply:
> Guess we can remove that note and we probably could now make dxvk the default.

My opinion is that we can use DXVK for Proton by default *when d3dcompiler_47 issue is fixed*.

Currently D3D11 rendering backend doesn't work without native DLL and I think that it would be *confusing for new users* if DXVK is used by default when using Proton.

Another reason is that DXVK requires Vulkan-capable GPU. "D3D11 compatible but non Vulkan compatible" GPUs may work with OpenGL backend (but I don't have such GPUs and can't test them).

Reply:
> Currently D3D11 rendering backend doesn't work without native DLL and I think that it would be confusing for new users if DXVK is used by default when using Proton.

Well, we would automatically install that dll. Sadly that dll is often used so it's one of the more common fixes to download that dll. Here's the status of the native implementation https://source.winehq.org/WineAPI/d3dcompiler_47.html

> Another reason is that DXVK requires Vulkan-capable GPU. "D3D11 compatible but non Vulkan compatible" GPUs may work with OpenGL backend (but I don't have such GPUs and can't test them).

Imho such ancient things shouldn't hold back improvements. Even my 660 from 2013 is capable of running Vulkan stuff. Older cards may not even have the performance to run the game in a reasonable way.